tests/integration/presto.spec.ts

```diff
@@ -0,0 +1,64 @@
+import { expect } from "chai";
+
+import * as dfapi from "df/api";
+import * as dbadapters from "df/api/dbadapters";
+import { dataform } from "df/protos/ts";
+import { suite, test } from "df/testing";
+import { compile, keyBy } from "df/tests/integration/utils";
+import { PrestoFixture, prestoTestCredentials } from "df/tools/presto/presto_fixture";
+
+suite("@dataform/integration/presto", { parallel: true }, ({ before, after }) => {
+  const _ = new PrestoFixture(1234, before, after);
```

Comment:
`1234` should reference the port set in the presto fixture file, instead of being hardcoded here